src/extension.ts

```diff
@@ -1309,20 +1309,133 @@ export async function activate(context: vscode.ExtensionContext) {
 	context.subscriptions.push(
 		vscode.commands.registerCommand(
 			'intuita.acceptFolder',
-			async (arg0, ...otherArgs) => {
-				const firstJobHash: string | null =
-					typeof arg0 === 'string' ? arg0 : null;
-				if (firstJobHash === null) {
-					throw new Error(
-						'Did not pass the jobHashes into the command.',
+			async (_arg0) => {
+				try {
+					const arg0: {
```

Comment:
you can actually use a `io-ts` codec for that one

Reply:
addressed